### PR TITLE
Add entry for awesome-zsh-plugins

### PR DESCRIPTION
**Pull request by unixorn**
List of oh-my-zsh compatible plugins for [oh-my-zsh](https://github.com/robbyrussell/oh-my-zsh), [zgen](https://github.com/tarjoilija/zgen) and [antigen](https://github.com/zsh-users/antigen)

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -178,6 +178,7 @@
 - [Sketch](https://github.com/diessica/awesome-sketch) *(OS X app)*
 - [Slack](https://github.com/filipelinhares/awesome-slack) *(Communities powered by Slack)*
 - [Podcasts](https://github.com/cv/awesome-geek-podcasts)
+- [Zsh plugins](https://github.com/unixorn/awesome-zsh-plugins) *(oh-my-zsh compatible plugins for [oh-my-zsh](https://github.com/robbyrussell/oh-my-zsh), [zgen](https://github.com/tarjoilija/zgen) and [antigen](https://github.com/zsh-users/antigen))*
```
Comment (sindresorhus) [inline readme.md]: I don't think the description is needed.

`Zsh plugins` => `ZSH Plugins`

**Comment by unixorn** [comment]
Ok, changed case and removed description

**Comment by sindresorhus** [comment]
Would be cool if you could link back here in your list so users can easily discover more awesome lists ;)

**Comment by unixorn** [comment]
Done.

**Pull request closed without merging**